app/assets/stylesheets/application.css

```diff
@@ -1,15 +0,0 @@
-/*
```

Comment:
Out of curiosity, why did we switch from css to scss?

Reply:
Sass uses scss. We could use css, but scss is more powerful/I really wanted to try it. Here's a link to a good explanation of why I chose it over flat css: [https://alistapart.com/article/why-sass](here)